spec/function.dd

```diff
@@ -2884,10 +2884,60 @@ $(H3 $(LNAME2 safe-functions, Safe Functions))
         so they can be corrected.
         )
 
+$(H4 Safe External Functions)
+
+        $(P External functions don't have a function body visible to the compiler:
+        ---
+        @safe extern (C) void play();
+        ---
+        and so safety cannot be verified automatically.
+        )
```

Comment:
Don't put code blocks into `$(P ...)`. HTML doesn't allow it. The empty second paragraph in the output is because of this.

Reply:
ok